ethicml/evaluators/evaluate_models.py

```diff
@@ -206,6 +213,7 @@ def evaluate_models(
                 train = train.get_n_samples()
             train = train.replace(name=f"{train.name} ({split_id})")
             data_splits.append(TrainTestPair(train, test))
+            model_seeds.append(0 if repeat_on == "data" else split_id)
```

Comment:
maybe we should store the model seeds in the `TrainTestPair`? that might be an easier way to get the information where it needs to go

Reply:
ah no, I see that `TrainTestPair` is also used in the tests where it doesn't make sense to have a seed